### DISCUSSION TITLE
custom domains?

### DISCUSSION THREAD

**Discussion by FrenchFryInteractive** [pockethost/pockethost#228] [Q&A] [answered]
Hey, just a newbie here. Are customs domains something i can do?

**Comment by benallfree** [comment, marked as answer]
They are in planning. #25 covers this question in detail.

**Reply by FrenchFryInteractive** [reply]
Amazing thanks. =)

**Reply by BasToTheMax** [reply]
Any updates? Would love to see this :D

**Reply by benallfree** [reply]
@BasToTheMax How would you use a custom domain? Would you deploy your frontend to pb_public?

**Reply by BasToTheMax** [reply]
I would use it as `api.example.com` with my frontend hosted on nging on a vps.

**Reply by benallfree** [reply]
One complicating factor is that SSL requires a dedicated IP address. I could direct users to proxy through a service like Cloudflare, where they would handle provisioning SSL certificates and allocating IP addresses. That might be the solution for now. Otherwise, I don’t have a free solution for SSL and custom domain names. Thoughts?

**Reply by benallfree** [reply]
Update: Cloudflare (hosting pockethost.io) supports Custom Hostnames now https://developers.cloudflare.com/cloudflare-for-platforms/cloudflare-for-saas/. We are good 👍
